src/repositories/user.repository.ts

```diff
@@ -8,4 +12,21 @@ export abstract class UserRepository {
       }
     });
   }
+
+  static findByGoogleId(googleId: string) {
+    return prisma.user.findUnique({
+      where: {
+        googleId
+      }
+    });
+  }
+
+  static createUser({ googleId, email }: UserGoogleParams) {
+    return prisma.user.create({
```

Comment:
As https://github.com/ledevthang/evocean-be/pull/3/files#r1590203813.
Let's use upsert prisma method instead of create. Create if not existed or just find user with given googleId.